src/test/java/org/openmrs/contrib/qaframework/automation/ManageUserAccountSteps.java

```diff
@@ -0,0 +1,136 @@
+package org.openmrs.contrib.qaframework.automation;
+
+import java.util.List;
+
+import io.cucumber.java.After;
+import io.cucumber.java.Before;
+import io.cucumber.java.en.And;
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+
+import org.openmrs.contrib.qaframework.RunTest;
+import org.openmrs.reference.page.ManageUserAccountPage;
+import org.openmrs.reference.page.SystemAdministrationPage;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+public class ManageUserAccountSteps extends Steps {
+
+	private static final String SYSTEM_ALERT = "Atleast 8 character(s) are required";
+	private SystemAdministrationPage systemAdministrationPage;
+	private ManageUserAccountPage userAccountPage;
+
+	@Before(RunTest.HOOK.SELENIUM_USER_ACCOUNT)
+	public void visitHomePage() {
+		initiateWithLogin();
+	}
+
+	@After(RunTest.HOOK.SELENIUM_USER_ACCOUNT)
+	public void destroy() {
+		quit();
+	}
+
+	@Given("a user clicks on System Administartion app from home page")
+	public void loadSystemAdministrationPage() {
+		systemAdministrationPage = homePage.goToSystemAdministrationPage();
+	}
+
+	@Then("the system loads the system administration page")
+	public void systemloadsSystemAdministrationPage() {
+		assertTrue(textExists("System Administration"));
+	}
+
+	@When("a user clicks on Manage accounts app")
+	public void loadManageAccountsPage() {
+		userAccountPage = systemAdministrationPage.goToManageAccounts();
+	}
+
+	@Then("the system loads the manage acccount page")
+	public void systemloadsManageAccountsPage() {
+		assertTrue(textExists("Manage Accounts"));
+	}
+
+	@When("a user clicks Add New Account button")
+	public void goToAddNewAccountPage() {
+		userAccountPage.clickOnAddUser();
+	}
+
+	@Then("the system loads the add new account form")
```

Comment:
Well done ,Would you mind to collect this gherkin name scenario to something more readable, something like   "**the system loads add new account form**".

Reply:
I agree!